src/components/Task.js

```diff
@@ -0,0 +1,11 @@
+const Task = ({value}) => {
+    return (<div>
+        <li className='list-item'>
+            <input type='checkbox'></input>
+            <span>{value}</span>
+            <button>x</button>
+        </li>
+    </div>)
```

Comment:
pay attention to code-style
https://github.com/airbnb/javascript/tree/master/react